UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs

```diff
@@ -182,7 +182,7 @@ public void ImplantPeriodicUpdate()
 			if(IsBleeding)
 			{
 				InternalBleedingLogic();
-				HealthMaster.CirculatorySystem.Bleed(limbLossBleedingValue);
+				//HealthMaster.CirculatorySystem.Bleed(limbLossBleedingValue); causes you to continue bleeding after hitting 0 bleedstacks
```

Comment:
please don't comment out code, remove it.

Reply:
gotcha